web/src/shared/components/Header/Search.tsx

```diff
@@ -169,6 +175,8 @@ const Searchbar: React.FunctionComponent = () => {
                 // alert("Family page not implemented yet");
                 navigate(`/family/${id}`)
                 break
+            case 'sequencing-group':
+                navigate(`/sample/${id}/${seqID}`)
```

Comment:
I'm a little nervous routes aren't formatted by some singular helper.

Reply:
I think this is pretty clear but can turn into some utility if you prefer - pass a type and id(s) and it'll navigate appropriately?

Reply:
I guess it's fine for now, just don't like that route logic can be scattered across the application. I'd prefer a utility that you pass the type and ID and returns the constructed route (where this class performs the navigate, or alternatively it's used in a Link).

Reply:
I'll add this to the to-do list